.github/workflows/push_rockspec.yml

```diff
@@ -33,15 +33,13 @@ jobs:
 
       # https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions
       - name: Set env
-        run: echo "GIT_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+        run: echo "GIT_TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
```

Comment:
According to the [documentation](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) (and the `startWith()` operator above) the variable is like `refs/tags/X.Y.Z`. So both variants should work:

Bash:

```shell
$ (GITHUB_REF=refs/tags/X.Y.Z; echo ${GITHUB_REF#refs/*/})
X.Y.Z
$ (GITHUB_REF=refs/tags/X.Y.Z; echo ${GITHUB_REF##*/})
X.Y.Z
```

Dash:

```shell
$ (GITHUB_REF=refs/tags/X.Y.Z; echo ${GITHUB_REF#refs/*/})
X.Y.Z
$ (GITHUB_REF=refs/tags/X.Y.Z; echo ${GITHUB_REF##*/})
X.Y.Z
```

So what is the motivation of the change? To follow usual pattern from other modules? Please, clarify it somewhere (commit message, pull request description).